src/pather.py

```diff
@@ -501,7 +503,7 @@ def _convert_rel_to_abs(rel_loc: tuple[float, float], pos_abs: tuple[float, floa
         return (rel_loc[0] + pos_abs[0], rel_loc[1] + pos_abs[1])
 
     def traverse_nodes_fixed(self, key: str | list[tuple[float, float]], char: IChar) -> bool:
-        if not char.capabilities.can_teleport_natively:
+        if SkillName.Teleport not in hotkeys.right_skill_key_map:
```

Comment:
Have you tested this behavior with charged amulet users? Fixed nodes require teleport in that they sometimes traverse walls. What happens if charged user runs out of charges during traverse? If can_teleport_natively works here still, then I'd leave this piece alone for this PR.